core/api/dev/apollo-federation/supergraph.graphql

```diff
@@ -1960,6 +1972,25 @@ interface Wallet
     paymentHash: PaymentHash!
   ): Invoice!
   pendingIncomingBalance: SignedAmount!
+
+  """
+  Pending OnChain transactions. When transactions
+  are confirmed they will receive a new id and be found in the transactions
+  list. Transactions are ordered anti-chronologically,
+  ie: the newest transaction will be first
+  """
+  pendingTransactions: [Transaction!]!
+
+  """
+  Pending OnChain transactions. When transactions
+  are confirmed they will receive a new id and be found in the transactions
+  list. Transactions are ordered anti-chronologically,
+  ie: the newest transaction will be first
+  """
+  pendingTransactionsByAddress(
```

Comment:
should not this be a filter of `pendingTransactions` ? cc @vindard @bodymindarts @nicolasburtey

Reply:
@bodymindarts reviewed this comment over standup and believes we should continue with this pattern